generators/app/tasks/appSetup/packageJsonScripts.js

```diff
@@ -5,9 +5,9 @@ module.exports = function packgeJsonScripts() {
   packageJson.scripts.ios = 'react-native run-ios --scheme qa';
   packageJson.scripts.test = 'jest';
   packageJson.scripts.coverage = 'jest --coverage';
-  packageJson.scripts.clean = 'rm -rf $TMPDIR/react-* && watchman watch-del-all && npm cache clean';
+  packageJson.scripts.clean = 'rm -rf $TMPDIR/react-* && watchman watch-del-all && yarn cache clean';
   packageJson.scripts['force-clean'] =
-    'yarn run android:clean && rm -rf $TMPDIR/react-* && watchman watch-del-all && rm -rf ios/build && rm -rf ios/Pods && rm -rf node_modules/ && npm cache clean && yarn install && cd ios/ && pod install';
+    'yarn run android:clean && rm -rf $TMPDIR/react-* && watchman watch-del-all && rm -rf ios/build && rm -rf ios/Pods && rm -rf node_modules/ && yarn cache clean && yarn install && cd ios/ && pod install';
```

Comment:
You could optimize this line using the `clean` script: `yarn run android:clean && yarn run clean && rm -rf ios/build && rm -rf ios/Pods && rm -rf node_modules/ && yarn install && cd ios/ && pod install"`